analysis/deep2_r23_o32/archives/binning/binning_and_graphing_mastergrid.py

```diff
@@ -9,22 +9,24 @@
 def n_times_binned(fitspath, pdf_pages, outfile, n_split, individual_ID, R23, O32,
                    SNR3, data3, galinbin):
     """
-    R23 and O32 are going to be log values
-    One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
-    increase the number of galaxies as R23 increases
+    Purpose:
+      R23 and O32 are going to be log values
+      One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
+      increase the number of galaxies as R23 increases
 
-    This file holds the function to bin data adaptively based on the entered number of galaxies for each bin.
-    First, it bins in the R23 direction and then in the O32 direction
-    Used in current analysis
+      This file holds the function to bin data adaptively based on the entered number of galaxies for each bin.
+      First, it bins in the R23 direction and then in the O32 direction
+      Used in current analysis
 
     Inputs:
```

Comment:
Parameters
------------

Reply:
I created #64 for this comment. Will address outside this PR.